tests/pw/pages/basePage.ts

```
@@ -1425,10 +1426,15 @@ export class BasePage {
     }
 
-    // multiple elements to be visible
-    async multipleElementVisible(selectors: any) {
+    async multipleElementVisible(selectors: { [key: string]: any }) {
         for (const selector in selectors) {
-            await this.toBeVisible(selectors[selector]);
+            if (helpers.isPlainObject(selectors[selector])) {
+                await this.multipleElementVisible(selectors[selector]);
+            } else if (typeof selectors[selector] === 'function') {
+                continue;
+            } else {
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)